.github/workflows/ci.yaml

```diff
@@ -0,0 +1,279 @@
+
+            - name: Composer - Set cache
+              uses: actions/cache@v2
+              id: cache-composer
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: php-${{ matrix.php }}-sylius-${{ matrix.sylius }}-symfony-${{ matrix.symfony }}-composer-${{ hashFiles('**/composer.json') }}
+                  restore-keys: php-${{ matrix.php }}-sylius-${{ matrix.sylius }}-symfony-${{ matrix.symfony }}-composer-
+
+            - name: Composer - Create cache directory
+              run: mkdir -p /home/runner/.composer/cache
+              if: steps.cache-composer.outputs.cache-hit != 'true'
+
+            - name: Composer - Self Update
+              run: sudo composer self-update --1
+              if: matrix.sylius == '1.7.*'
+
+            - name: Composer - Github Auth
+              run: composer config -g github-oauth.github.com ${{ github.token }}
+
+            - name: Composer - Install Sylius-Standard
+              run: composer create-project --no-progress sylius/sylius-standard sylius "${{ matrix.sylius }}"
+
+            - name: Composer - Add path repository
+              working-directory: ./sylius
+              run: |
+                  composer config repositories.plugin '{"type": "path", "url": "../plugin/"}'
+
+            - name: Composer - Config
+              working-directory: ./sylius
+              run: |
+                  composer config extra.symfony.allow-contrib true
+                  composer config minimum-stability "dev"
+                  composer config prefer-stable true
+
+            - name: Composer - Restrict Symfony version
+              working-directory: ./sylius
+              run: composer config extra.symfony.require "${{ matrix.symfony }}"
+
+            - name: Composer - Install plugin
+              working-directory: ./sylius
+              run: composer req "${{ env.package-name }}:*" --prefer-source --no-scripts
+
+            - name: Sylius - Update dependencies # need to update sylius version
+              working-directory: ./sylius
+              run: |
+                  composer update --no-progress --no-scripts --prefer-dist
+                  composer update --no-suggest -n -o
+
+            - name: Sylius - Run Symfony flex
+              working-directory: ./sylius
+              run: composer sync-recipes
+
+            - name: Sylius - Copy migration of Plugin
+              working-directory: ./sylius
+              run: cp ../plugin/src/Migrations/* src/Migrations
+              if: matrix.sylius == '1.7.*'
+
+            - name: Sylius - Migration config
+              working-directory: ./sylius
+              run: |
+                  rm src/Migrations/*
+                  echo "        'Synolia\SyliusSchedulerCommandPlugin\Migrations': '@SynoliaSyliusSchedulerCommandPlugin/Migrations'" >> config/packages/doctrine_migrations.yaml
```

Comment:
What about automatically register it ?  Or you should at least update README part.

Reply:
No mention of it in Readme, but i need to update Flex recipe (https://github.com/symfony/recipes-contrib/blob/a8e030d0cb61f872021c422558ecf777f51bf752/synolia/sylius-scheduler-command-plugin/1.0/manifest.json#L9)

Reply:
It's mentionned in readme with the old style : 
![image](https://user-images.githubusercontent.com/3168281/114182697-38383b80-9943-11eb-99f4-78a1cc6f21a1.png)
You are right, this recipe should also get updated. 

And, I think you can automatically register it without altering `config/packages/doctrine_migrations.yaml` by using trait `PrependDoctrineMigrationsTrait` provided by sylius, (or using same logic if we have to support older version that not contains this trait)

Reply:
Like that https://github.com/Sylius/RefundPlugin/blob/817ff65aa2728b34d14a9eb70b3fea0f2efb2760/src/DependencyInjection/SyliusRefundExtension.php ?

Reply:
Thas was introduce in Sylius v1.8.6, so i think i will drop support of Sylius 1.7 ;)
I'll do that in next PR, i'll merge this one.

Reply:
Job done 😄